docs/Components/Aerodynamic/RectangularHorizontalStabilizer.md

```diff
@@ -0,0 +1,19 @@
+## RectangularHorizontalStabilizer
+This class creates a rectangular horizontal stabilizer
+```python
+from vec import Vector2
+import math
+import pytest
+from adr.Components.Aerodynamic import RectangularHorizontalStabilizer
+
+rectangularHorizontalStabilizer = RectangularHorizontalStabilizer(
+        relative_position=Vector2(x=-0.7, y=0.2),
+        relative_angle=math.radians(0),
+        inverted=True,
```

Comment:
inverted=True is not needed. The HorizontalStabilizer class automatically add this one.